### PR TITLE
Rename Schema => SchemaController

### PR DESCRIPTION
**Pull request by drew-gross**
Also fix an error that snuck in during merge

### CODE REVIEW

**Comment by flovilmart** [comment]
In the controllers folder?

**Comment by drew-gross** [comment]
Ya will do

File: src/Controllers/DatabaseController.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import intersect from 'intersect';
 var mongodb = require('mongodb');
 var Parse = require('parse/node').Parse;
 
-var Schema = require('./../Schema');
+var SchemaController = require('./../Controllers/SchemaController');
```
Comment (flovilmart) [inline src/Controllers/DatabaseController.js]: We can remove leading `./`

File: src/Controllers/SchemaController.js [hunk 2]
```diff
@@ -15,8 +15,7 @@
 // TODO: hide all schema logic inside the database adapter.
 
 const Parse = require('parse/node').Parse;
-import MongoSchemaCollection from './Adapters/Storage/Mongo/MongoSchemaCollection';
```
Comment (flovilmart) [inline src/Controllers/SchemaController.js]: :+1:

**Comment by flovilmart** [comment]
Baby nit, otherwise LGTM

**Pull request merged**